wgsl/index.bs

```diff
@@ -9390,18 +9422,32 @@ Each input-output location can store a value up to 16 bytes in size.
 The byte size of a type is defined using the *SizeOf* column in [[#alignment-and-size]].
 For example, a four-component vector of floating-point values occupies a single location.
 
-IO locations are specified via the [=attribute/location=] attribute.
-
 Each user-defined [=user-defined input datum|input=] and [=user-defined output datum|output=] [=shader-creation error|must=] have an explicitly specified IO location.
 Each structure member in the entry point IO [=shader-creation error|must=] be one of either a built-in value
 (see [[#builtin-inputs-outputs]]), or assigned a location.
 
+A location can be specified via either the [=attribute/location=] attribute only or both the
```

Comment:
This is still feels unlcear to me. Maybe instead let's drop this paragraph and integrate it into the overlap rules.